bindata/etcd/restore-pod.yaml

```diff
@@ -24,6 +24,43 @@ spec:
         export ETCD_NAME=${NODE_NODE_ENVVAR_NAME_ETCD_NAME}
         export ETCD_INITIAL_CLUSTER="${ETCD_NAME}=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380"
         env | grep ETCD | grep -v NODE
+        export ETCD_NODE_PEER_URL=https://${NODE_NODE_ENVVAR_NAME_ETCD_DNS_NAME}:2380
+
+        # checking if data directory is empty, if not etcdctl restore will fail
+        if [ -d "/var/lib/etcd " ]; then
+          echo "please delete the data directory before restoring"
+          exit 1
```

Comment:
This won't work on a pod. "/var/lib/etcd" is a mount point, and it will always be there. :-(

Reply:
restored to ls -A 👍